README.rst

```diff
@@ -1,23 +1,87 @@
 Running Cyclus on Blue Waters
 =============================
 
```

Comment:
Please add some explanatory text here. Specifically, set up an example use case. "Imagine you have a cyclus simulation that you would like to run many times with varied parameters or agents. You can use this workflow to generate an input file for each desired simulation and to run those input files simultaneously on the Blue Waters supercomputer."